ILSpy/App.xaml

```diff
@@ -6,19 +6,19 @@
              xmlns:ilSpy="clr-namespace:ICSharpCode.ILSpy"
              xmlns:themes="clr-namespace:ICSharpCode.ILSpy.Themes"
              StartupUri="MainWindow.xaml">
-	<Application.Resources>
-		<Style x:Key="DialogWindow" TargetType="{x:Type Window}">
-			<Setter Property="ShowInTaskbar" Value="False"/>
-			<Setter Property="UseLayoutRounding" Value="True"/>
-			<Setter Property="TextOptions.TextFormattingMode" Value="Display"/>
-			<Setter Property="toms:StyleBindings.Behaviors">
-				<Setter.Value>
-					<toms:BehaviorCollection>
-						<themes:WindowStyleManagerBehavior />
-					</toms:BehaviorCollection>
-				</Setter.Value>
-			</Setter>
-		</Style>
+    <Application.Resources>
```

Comment:
It's hard to review if you completely reformat whole files. (Applies to many others, too)

Reply:
If you append `?w=1` to the GitHub URL, e.g. https://github.com/icsharpcode/ILSpy/pull/3234/files?w=1 then you can review everything without whitespace changes.

Reply:
But I agree that this should be reverted, especially because the only change in this file is the indentation.